### PR TITLE
Add content on custom tracing port

### PR DESCRIPTION
**Pull request by shavidissa**
Add content on custom tracing port.

### CODE REVIEW

**Comment by shavidissa** [comment]
CC @srujann 

The doc link is broken for now until the changes get pushed to mater.

File: README.md [hunk 1]
```diff
@@ -135,7 +135,9 @@ You can send metrics, histograms, or trace data from your application to the Wav
 [Create a DirectConfiguration](#option-2-sending-data-via-direct-ingestion) to send data directly to a Wavefront service.
   
 ### Option 1: Sending Data via the Wavefront Proxy
-Depending on the data you wish to send to Wavefront (metrics, distributions (histograms) and/or spans), enable the relevant ports on the proxy and initialize the proxy sender as follows:
+Depending on the data you wish to send to Wavefront (metrics, distributions (histograms) and/or spans), enable the relevant ports on the proxy and initialize the proxy sender.
+
+When you use a Sender SDK, you won’t see span-level RED metrics by default unless you use the Wavefront proxy and define a custom tracing port (`tracingPort`). See [Instrument Your Application with Wavefront Sender SDKs](https://www.wavefront.com/tracing_instrumenting_frameworks.html/#instrument-your-application-with-wavefront-sender-sdks) for details.
```
Comment (srujann) [inline README.md:140]: add a subsection title that this is related to sending spans.

Comment (shavidissa) [reply README.md]: Thank you! There was a tracing span subsection. I moved it there.

File: README.md [hunk 2]
```diff
@@ -266,6 +265,8 @@ sender.SendDistribution("request.latency", centroids, hgs, 0, "appServer1", map[
 
 #### Tracing Spans
 
+When you use a Sender SDK, you won’t see span-level RED metrics by default unless you use the Wavefront proxy and define a custom tracing port (`TracingPort`). See [Instrument Your Application with Wavefront Sender SDKs](https://www.wavefront.com/tracing_instrumenting_frameworks.html/#instrument-your-application-with-wavefront-sender-sdks) for details.
```
Comment (vikramraman) [inline README.md:268]: the given link doesn't work?

Comment (shavidissa) [reply README.md]: Thank you! Fixed. :)

**Pull request merged**